app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] I highly recommend making the wireframe for login the same as the given wireframe since in the given one the width for all the inputs is the same and it should be the same so please fix this as well in both the login form and signup form that would be amazing 👏🏻🔥💪🏻♻

- Your design
![image](https://github.com/Ochiengsteven/art-post/assets/58553711/04a33d49-3886-44eb-8adf-95db7a49a4cb)


- Given
![image](https://github.com/Ochiengsteven/art-post/assets/58553711/ed031123-04cd-4f2f-b5e5-6882f6133fe0)